packages/runtime/package.json

```diff
@@ -31,6 +32,7 @@
     "lodash-es": "^4.17.21",
     "react": "^18.2.0",
     "react-chartjs-2": "^5.2.0",
+    "react-lottie": "^1.2.3",
```

Comment:
This package is over 5 years old, maybe consider this one https://www.npmjs.com/package/lottie-react